### ISSUE TITLE
Persistent Customer Form

### ISSUE THREAD

**Issue by Escapada-Eric** [magento-hackathon/HoneySpam#12]
Hi,
We're using mage_persistent for our customer account creation. Is there an easy way to configure this to work for that?

XXX/XXX/template/PERSISTENT/customer/form/register.phtml

**Comment by Schrank** [comment]
I would assume it does work. Have you tested it? Did you clean the cache? Does it work on any other form?

**Comment by Escapada-Eric** [comment]
Yep, I've cleared the cache and checked the other usual suspects. It looks like this isn't working on our other forms either. Looking at the template hints in Magento, I am not seeing honeypot.phtml being loaded on the account creation screen. Just that register.phtml from the persistent template.

It could be something with our Magento implementation but just wanted to check if anyone else has run into this using persistent.
